serverless/src/submitRetrieval/__tests__/handler.test.js

```diff
@@ -140,6 +141,14 @@ describe('submitRetrieval', () => {
         query.response([{
           id: 5
         }])
+      } else if (step === 7) {
+        query.response([{
+          id: 5
```

Comment:
We already talked about this but I just want to remind myself Monday, these IDs need to be incremented in each response, and the `MessageBody`s below adjusted accordingly

Reply:
I pushed to the wrong repo.